interfaces/powermeter.yaml

```diff
@@ -26,5 +26,8 @@ vars:
     description: Measured dataset
     type: object
     $ref: /powermeter#/Powermeter
+  public_key:
```

Comment:
In the mean time this was added in core as well but with a different name: public_key_ocmf